fastlane/Fastfile

```diff
@@ -70,6 +70,37 @@ platform :android do
     )
   end
 
+  desc "Tag amazon-latest release version with latest if necessary"
```

Comment:
these are just duplicates of the other lanes but tagging amazon-latest. This will not be merged to main or anything and it's just for these alpha releases